src/platform/plugins/shared/discover/public/application/main/components/top_nav/use_discover_topnav.ts

```diff
@@ -34,6 +35,7 @@ export const useDiscoverTopNav = ({
   const services = useDiscoverServices();
   const topNavCustomization = useDiscoverCustomization('top_nav');
   const hasUnsavedChanges = useInternalStateSelector((state) => state.hasUnsavedChanges);
+  const isMobile = useIsWithinBreakpoints(['xs']);
```

Comment:
```suggestion
  const isMobile = useIsWithinMaxBreakpoint('xs');
```
If we use `useIsWithinMaxBreakpoint` instead, it would still work if we added an even smaller breakpoint in the future like `xxs`.

But it doesn't have to change since I doubt that would actually happen or lots of other places would break too, just sharing a tip.